### PR TITLE
[bitnami/harbor] Add variable to assign postgres password via an existing secret

### PR DESCRIPTION
**Pull request by lusu007**
**Description of the change**

This is especially useful to use if the values.yaml files are versioned in Git. Files versioned in Git should never contain passwords.

**Benefits**

Values.yaml does not have to contain a password.

**Possible drawbacks**

Nothing as far as I know.

**Applicable issues**

#5468 

**Additional information**

If you need more information on this topic just write a comment.

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/harbor/Chart.yaml [hunk 1]
```diff
@@ -34,4 +34,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-harbor-registry
   - https://github.com/bitnami/bitnami-docker-harbor-registryctl
   - https://goharbor.io/
-version: 9.4.6
+version: 9.4.7
```
Comment (Mauraza) [inline bitnami/harbor/Chart.yaml:37]: Could you change this to a minor versión?
```
version: 9.5.0
```

File: bitnami/harbor/values.yaml [hunk 2]
```diff
@@ -2685,6 +2685,10 @@ postgresql:
   nameOverride:
   postgresqlUsername: postgres
   postgresqlPassword: not-secure-database-password
+  ## Name of existing secret to use for PostgreSQL passwords.
+  ## If an existingSecret is used it will overwrite postgresqlUsername and postgresqlPassword.
+  ## The secret has to contain at least the key postgresql-postgres-password.
```
Comment (Mauraza) [inline bitnami/harbor/values.yaml:2690]: Could you add an example of the secret?
```
# apiVersion: v1
# kind: Secret
# metadata:
#   name: mysecret
# type: Opaque
# data:
#   postgresqlUsername: YWRtaW4=
#   postgresqlPassword: MWYyZDFlMmU2N2Rm
```

**Review by Mauraza — COMMENTED** [review]
Hi @lusu007,

thanks for the contribution! You add the new variable but you don't change the templates to get the secret. Could you add too and check two small comments?

**Comment by lusu007** [comment]
Hey @Mauraza ,
I just resolved your two comments.

But I don't understand what you mean with
> You add the new variable but you don't change the templates to get the secret. 

The secret is passed through the Harbor Helm Chart into to the postgres dependency. isn't that enough? what else do i need to adjust?

**Comment by Mauraza** [comment]
Hi @lusu007,

> Do I need to remove the changes in Chart.lock file?

yes, please.

Could you do a rebase master to resolve de conflicts? 
And with that is ready to merge.

Thank again for the contribution!

**Comment by lusu007** [comment]
Hey @Mauraza,
The PR should now be ready to merge. 😄

> Thank again for the contribution!
Gladly again and again.

**Comment by Mauraza** [comment]
LGTM!

**Pull request merged**